OpenRA.Platforms.Default/Sdl2PlatformWindow.cs

```diff
@@ -26,122 +27,159 @@ sealed class Sdl2PlatformWindow : ThreadAffine, IPlatformWindow
 		internal readonly IntPtr Window;
```

Comment:
I think this technically needs to be covered by the `syncObject` too since it will be read from the render thread.

Reply:
Really? This is initialized before the renderer thread is created, and doesn't change.

Reply:
I'm not familiar enough with the threading model to *prove* reading it in this specific instance without any synchronisation is safe - even though I wouldn't be surprised if your reasoning is indeed accurate.

Unless I can prove something is safe - I'd prefer to just take a lock and remove any doubt. It doesn't look likely to be a problem for performance here.

Reply:
I agree, better safe than sorry.